templates/dashboard/password_update.html

```diff
@@ -1,19 +1,53 @@
 {% extends 'base.html' %}
 
-{% block title %}Change Password{% endblock %}
+{% block title %}
+  Change Password
+{% endblock %}
 
 {% block content %}
-<h2>Change Password</h2>
-{% if messages %}
-    <ul class="messages">
+  <h2>Change Password</h2>
+  {% if messages %}
```

Comment:
This block could be simple without nesting it. Taking advantage of Jinja template options like default. 

```
{% if messages %}
    <div class="messages">
        {% for message in messages %}
            <div class="alert alert-dismissible alert-{{ message.tags|default:"info" }}">
                <strong>{{ message.tags|capfirst }}:</strong> {{ message }}
            </div>
        {% endfor %}
    </div>
  {% endif %}
```